internal/mage/util/engine.go

```diff
@@ -0,0 +1,327 @@
+package util
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"runtime"
+	"sort"
+	"text/template"
+
+	"dagger.io/dagger"
+	"golang.org/x/exp/maps"
+)
+
+const (
+	engineBinName = "dagger-engine"
+	shimBinName   = "dagger-shim"
+	alpineVersion = "3.17"
+	runcVersion   = "v1.1.4"
```

Comment:
`runcVersion` is still in `internal/mage/engine.go`, but it's at `v1.1.5`, should probably remove it there and update here to the latest version